Articles/WhatIs/WhatAreConferencesAndWorkshops.md

```diff
@@ -0,0 +1,28 @@
+### What are Conferences and Workshops for Better Scientific Software?
+#### Contributed by  [Rinku Gupta](https://github.com/rinkug)
+#### Publication date: April 20, 2023
+
+<!--- deck start --->
+Empowering researchers in scientifivc computing with knowledge-sharing, collaboration, and networking opportunities through conferences and workshops.
+<!--- deck end --->
+
+<!--- body start --->
+
+Conferences and workshops play a pivotal role in advancing scientific research in the field of scientific computing. These events offer researchers a valuable platform to showcase their latest findings, discuss and exchange ideas with other experts, and cultivate collaborations that drive innovation in their field. Beyond knowledge-sharing and collaboration, attending conferences and workshops enables researchers to remain up-to-date on the latest trends and advancements in their field, enabling them to refine their research approach and stay ahead of the curve.
+
+Conferences and workshops provide researchers with a unique opportunity to present their work to an audience of peers, fostering critical discussions that encourage the exchange of ideas and feedback. Through these platforms, researchers receive constructive criticism that enables them to fine-tune their ideas and approaches, resulting in the development of new insights, techniques, and approaches that can further advance their research agenda. Scientific computing software stacks are built for various applications from different domains. Conferences and workshops offer researchers a chance to connect with experts from other domains and fields, fostering interdisciplinary collaboration and knowledge-sharing. These events also provide excellent opportunities for industry, academia, national laboratories, and other research organizations to connect and exchange notes. Several of these factors result in awareness of new funding and grant opportunities. With both formal and informal settings, conferences/workshops are considered good environments to build deeper, meaningful relationships with one's colleagues and advance in the field.
```

Comment:
* platforms --> foums
* The sentence on research stacks seems misplaced...maybe remove? 
* Last few sentences seem a repeat of end of paragraph 1 with exception of awareness of *funding* too. I think most of this paragraph could be collapsed into single new sentence in paragraph 1.

Reply:
Sounds good